packages/Manager/src/main/resources/db/migration/V0017__water_allocations_table.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE water_allocations
+(
+  id                      INTEGER GENERATED BY DEFAULT AS IDENTITY,
+  area_id                 VARCHAR NOT NULL UNIQUE,
+  amount                  INTEGER NOT NULL,
+  last_updated_ingest_id  VARCHAR NOT NULL,
+  created_at              TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at              TIMESTAMPTZ NOT NULL,  
+  received_at             TIMESTAMPTZ NOT NULL,
```

Comment:
This [answer on Jooq date/time mapping](https://stackoverflow.com/questions/71576487/in-jooq-how-do-we-use-java-instant-to-map-to-postgresqls-timetamp-type/71734939#71734939) prompted me to check some [Postgres docs which reccommend against timestamo without tz](https://wiki.postgresql.org/wiki/Don%27t_Do_This#Don.27t_use_timestamp_.28without_time_zone.29). I appreciate the issue is more complex than a blanket recommendation, but I think they better model the data we are capturing, and help mitigate potential issues caused by app/server specific TZ settings.

Currently all other tables are simply using timestamp, so if we decide to go this way, I'd suggest we create a follow up card to convert them, and `updated_at` fields, and also add a Jooq converter.

Reply:
Yes I agree, need to program myself to remember that recommendation ... I appreciate what they are saying, and I keep defaulting to the "Use timestamp (without time zone) to store UTC times" and then suffering when the calling program doesn't do the same thing